src/test/java/org/takes/facets/auth/social/PsGithubTest.java

```diff
@@ -148,4 +176,29 @@ private static void assertParam(final Request req,
             Matchers.equalTo(value)
         );
     }
+
+    /**
+     * An inner class for the Take implementation testing.
+     */
+    private static class PsGithubTestTake implements Take {
```

Comment:
@sebing would also be a good idea to rename the class to `TkFakeLogin` or something similar.